polytope/polytope.py

```diff
@@ -772,21 +775,22 @@ def is_convex(reg, abs_tol=ABS_TOL):
     outer = envelope(reg)
     if is_empty(outer):
         # Probably because input polytopes were so small and ugly..
-        return False,None
+        return False, None
 
-    Pl,Pu = reg.bounding_box
-    Ol,Ou = outer.bounding_box
+    Pl, Pu = reg.bounding_box
+    Ol, Ou = outer.bounding_box
 
-    bboxP = np.hstack([Pl,Pu])
-    bboxO = np.hstack([Ol,Ou])
+    bboxP = np.hstack([Pl, Pu])
+    bboxO = np.hstack([Ol, Ou])
 
-    if sum(abs(bboxP[:,0] - bboxO[:,0]) > abs_tol) > 0 or \
-    sum(abs(bboxP[:,1] - bboxO[:,1]) > abs_tol) > 0:
-        return False,None
+    if sum(abs(bboxP[:, 0] - bboxO[:, 0]) > abs_tol) > 0 or \
```

Comment:
My proposal has the negative characteristic that the block indentation aligns with the internal expression of the condition. PEP 8 does not recommend one way or another for handling this; quoting:
>  This PEP takes no explicit position on how (or whether) to further visually distinguish such conditional lines from the nested suite inside the if -statement

Also we may wonder whether `or` should be moved to the first line of the conditional expression. I argue it is better as the first operator on the second line because, then, it is immediately clear that this is a disjunction.

However, I have no strong preference beyond avoiding the line continuation with `\`.